namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf

```diff
@@ -0,0 +1,23 @@
+module "ec-cluster-prison-visits-booking-staff" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.0"
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "${var.team_name}"
+  application            = "prison-visits-booking-staff"
```

Comment:
Could you please add : 
```
node_type = cache.m4.xlarge
```

Reply:
@mtrblmoj - added

Reply:
You've also got this failure in concourse: 

Error: module "ec-cluster-prison-visits-booking-staff": "aws_region" is not a valid argument

With this version of the module, you also need a providers block, as per:
https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/master/example/elasticache.tf

Reply:
@digitalronin - 🤞 the changes have worked